security/subsystem/src/main/java/org/jboss/as/security/lru/FastConcurrentDirectDeque.java

```diff
@@ -1494,7 +1630,7 @@ private boolean casTail(Node<E> cmp, Node<E> val) {
         NEXT_TERMINATOR = new Node<>();
         NEXT_TERMINATOR.prev = NEXT_TERMINATOR;
         try {
-            UNSAFE = getUnsafe();
+            UNSAFE = sun.misc.Unsafe.getUnsafe();
```

Comment:
This won't work unless the calling class is in the system domain (which it won't be).  You'll get a SecurityException.